css/tweets.css

```diff
@@ -0,0 +1,15 @@
+textarea {
+    width: 100%;
+}
+
+form, .tweets {
+    margin-bottom: 100px;
+}
+
+.tweets img {
+    height: 80px;
```

Comment:
use "%" instead of "px" where possible

Reply:
Disagree. For images, until I have a server side rescaling enforcing unified image size (and even then...), I believe it is best to apply that from the client. With pixels, images just look good in every format.